Sming/SmingCore/Network/Http/HttpRequestAuth.h

```diff
@@ -20,22 +20,25 @@ class HttpRequest;
 class AuthAdapter
 {
 public:
-	virtual void setRequest(HttpRequest* request) = 0;
-
-	__forceinline virtual void setResponse(HttpResponse* response)
+	virtual ~AuthAdapter()
 	{
-		return;
 	}
 
-	virtual ~AuthAdapter()
+	virtual void setRequest(HttpRequest* request) = 0;
+
+	virtual void setResponse(HttpResponse* response)
 	{
 	}
 };
 
 class HttpBasicAuth : public AuthAdapter
 {
 public:
-	HttpBasicAuth(const String& username, const String& password);
+	HttpBasicAuth(const String& username, const String& password)
```

Comment:
@mikee47 You can initialize the variables in initialization list

![image](https://user-images.githubusercontent.com/247452/52957341-bdda8500-3391-11e9-98f2-67d8a293c63d.png)